contracts/scripts/defender-actions/harvest.js

```diff
@@ -30,39 +30,54 @@ const handler = async (event) => {
   log(`Resolved OETH Harvester Simple address to ${harvesterAddress}`);
   const harvester = new ethers.Contract(harvesterAddress, harvesterAbi, signer);
 
-  const firstNativeStakingProxyAddress =
-    addresses[networkName].NativeStakingSSVStrategyProxy;
-  log(
-    `Resolved first Native Staking Strategy address to ${firstNativeStakingProxyAddress}`
-  );
-  await harvest(harvester, firstNativeStakingProxyAddress, signer, "first");
+  const convexAMOProxyAddress = addresses[networkName].ConvexOETHAMOStrategy;
 
-  const secondNativeStakingProxyAddress =
-    addresses[networkName].NativeStakingSSVStrategy2Proxy;
-  log(
-    `Resolved second Native Staking Strategy address to ${secondNativeStakingProxyAddress}`
-  );
-  await harvest(harvester, secondNativeStakingProxyAddress, signer, "second");
+  // Always harvest from Convex AMO
+  const strategiesToHarvest = [convexAMOProxyAddress];
+  const strategiesDesc = ["Convex AMO"];
```

Comment:
I'd prefer doing a `address -> label` mapping, something like:
```
const labels = {
[convexAMOProxyAddress]: "Convex AMO",
[addresses[networkName].NativeStakingProxy1]: "Staking Strategy 1",
...
}
```

then you can just `labels[address]` to get the label

Reply:
Could also have it global, so we don't have to pass it around functions